test/runtests.jl

```diff
@@ -14,61 +14,126 @@ function test_edf_signal(rng, label, transducer, physical_units,
-        _edf_signal("EEG F4-M1", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("EEG C3-M2", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("EEG O1-M2", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("C4-M1", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("O2-A1", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("E1", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("E2", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("Fpz", "E", "uV", -32768.0f0, 32767.0f0),
-        EDF.AnnotationsSignal(samples_per_record, anns_1),
-        _edf_signal("EMG LAT", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("EMG RAT", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("SNORE", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("IPAP", "", "cmH2O", -74.0465f0, 74.19587f0),
-        _edf_signal("EPAP", "", "cmH2O", -73.5019f0, 74.01962f0),
-        _edf_signal("CFLOW", "", "LPM", -309.153f0, 308.8513f0),
-        _edf_signal("PTAF", "", "v", -125.009f0, 125.009f0),
-        _edf_signal("Leak", "", "LPM", -147.951f0, 148.4674f0),
-        _edf_signal("CHEST", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("ABD", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("Tidal", "", "mL", -4928.18f0, 4906.871f0),
-        _edf_signal("SaO2", "", "%", 0.0f0, 100.0f0),
-        EDF.AnnotationsSignal(samples_per_record, anns_2),
-        _edf_signal("EKG EKGR- REF", "E", "uV", -9324.0f0, 2034.0f0),
-        _edf_signal("IC", "E", "uV", -32768.0f0, 32767.0f0),
-        _edf_signal("HR", "", "BpM", -32768.0f0, 32768.0f0),
-        _edf_signal("EcG EKGL", "E", "uV", -10932.0f0, 1123.0f0),
-        _edf_signal("- REF", "E", "uV", -10932.0f0, 1123.0f0),
-        _edf_signal("REF1", "E", "uV", -10932.0f0, 1123.0f0),
-    ]
+               EDF.TimestampedAnnotationList(i, 0, ["", "$i c", "$i d"])]
+              for i in 1:n_records]
+    _edf_signal = (label, transducer, unit, lo, hi) -> test_edf_signal(rng, label,
+                                                                       transducer, unit, lo,
+                                                                       hi, imin16,
+                                                                       imax16,
+                                                                       samples_per_record,
+                                                                       n_records, T)
+    edf_signals = Union{EDF.AnnotationsSignal,EDF.Signal{T}}[_edf_signal("EEG F3-M2", "E",
+                                                                         "uV", -32768.0f0,
+                                                                         32767.0f0),
+                                                             _edf_signal("EEG F4-M1", "E",
+                                                                         "uV", -32768.0f0,
+                                                                         32767.0f0),
```

Comment:
can you re-write this so that it passes without this awful line wrapping somehow?  maybe by making an alias for the type?

```julia
ST = Union{EDF.AnnotationSignal,EDF.Signal{T}}
```

it'll wrap some of them but this is really bad